pkg/lib/interaction/nodes/authn_oob.go

```diff
@@ -36,6 +37,7 @@ func (e *EdgeAuthenticationOOB) Instantiate(ctx *interaction.Context, graph *int
 			Code: input.GetOOBOTP(),
 		},
 	}, &facade.VerifyOptions{
+		Form: otp.FormCode,
```

Comment:
I'm not sure if it is always code here, as we support using link in oob authentication too.

Reply:
Those login link are handled in pkg/lib/linteraction/nodes/authn_login_link.go

Reply:
So this node is really for code only.